### DISCUSSION TITLE
Data directory is mounted multiple times according to the 'lsblk' command

### DISCUSSION THREAD

**Discussion by tijnsc** [nextcloud/all-in-one#3161] [Questions] [answered]
I have installed Nextcloud AIO on Ubuntu Server LTS 22.04. I have recently migrated my old raspberry pi nextcloud server database and data directory to a new machine. I have followed the instructions at https://github.com/nextcloud/all-in-one/blob/main/migration.md.

When I run the lsblk command to see what paths are mounted on my system I see the '/mnt/ncdata' path is mounted twice or sometimes three to five times to /dev/sdb1, which is part of the secondary internal drive in my system. It seems that every time the 'nextcloud-aio-nextcloud' docker container is run, another mount is added to this list. 
I have tried removing the automount rule for this partition from my /etc/fstab file, but when I start the nextcloud-aio-nextcloud the path '/mnt/ncdata' is mounted to the system drive, so something in the nextcloud-aio-nextcloud container is automatically mounting this directory.

I was wondering if this is a problem, and how I could fix this if it is.

**Comment by szaimen** [comment]
Why was it closed?

**Reply by tijnsc** [reply, marked as answer]
Hello szaimen, I have solved the problem by clearing my AIO install and creating a new instance with only the data directory copied to this new instance instead of both the data directory and database. I think the problem could have been caused by incorrectly editing the database dump to change the data directory.
My apologies, I should have mentioned that here before closing the thread.

**Reply by szaimen** [reply]
All right, thanks for the reply!
